docs/governance/steering/elections.md

```diff
@@ -0,0 +1,126 @@
+
+### Emeritus Term
+
+Members of the steering committee who complete a term will graduate to becoming Emeritus members of the steering committee upon vacating their seat. This confers honor on the recipient, acknowledging the significant contributions they have made to the project. Emeritus members have no binding vote, and no expectation of continued participation in steering committee affairs.
+
+## Election Schedule and Operation
+
+After the [bootstrap] election, the steering committee is responsible for the appointment of election officers to operate the election and circulate an exact election timeline to the community. The steering committee should provide the election officer with a desired timeline based on current community needs.
+
+For example:
+
+- End of July
+  - Election officers
+  - Voter eligibility criteria
+  - Election preparation
+- September   
+  - Nomination period and election
+- October  
+  - Conclusion of Election
+  - Results announced at first community meeting after the election concludes
+
+The election officers will choose exact dates for each step and propose the final schedule to the Steering Committee.
+
+### Nomination and Election Procedure
+
+Nominations will be made using the _Nomination_ GitHub Issue template.
+
+Elections will be held using an online preference election system which supports Condorcet elections. The most preferred candidates will be elected to the open seats.
+
+### Election Officer Eligibility
+
+The steering committee should choose multiple election officers using the following criteria, so as to promote healthy rotation and diversity:
+
+- Each individual must be eligible to vote.
+- Each individual must have been a FINOS member for at least one year.
+- At least one election officer should have served before.
+- At least one election officer should have never served before.
+- Each officer should come from a different employer.
+- Each officer can be relied upon to follow the [election procedure].  
+
+History of election officers:  
+
+|Year|Officers|
+|---|---|
+| 2024 | TBD |
+
+## Steering Committee and Election Officer Recusal
+
+Currently serving steering committee members and the appointed election officers
+pledge to recuse themselves from any form of electioneering, including
+campaigning, nominating, or endorsing. Violations of this pledge should lead to
+a vote of no confidence in the offending party.
+
+Steering committee members _may_ ask other contributors to consider running,
+and they _may_ vote, so long as this information is kept private.
+
+Steering committee members who intend to run for re-election _may_
+self-nominate but are otherwise expected to adhere to this recusal.
+
+## Attribution
+
+This document was adapted from the Kubernetes Steering Committee Elections document [[afb3858](upstream)].
```

Comment:
Same here - link isn't redirecting to https://github.com/kubernetes/steering/blob/afb3858/charter.md

Reply:
I think this is to do with the GitHub code review tool. If you click the ellipsis top right and "view file" then try the links, they work. Or, at least the ones I tested did.